projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeleteJob.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+
+export interface IDeleteJobDialogProps {
+    /**
+     * Current Job name
+     */
+    jobName: string;
+}
+
+
+export default class DeleteJobDialog extends Component<IDeleteJobDialogProps> {
+    /**
+   * Returns a React component for rendering a delete menu.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+    constructor(props: IDeleteJobDialogProps) {
+        super(props);
+    }
+    /**
+    * Renders a dialog for deleting data job.
+    *
+    * @returns React element
+    */
+    render(): React.ReactElement {
+        return (
+            <>
+                 <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobName">Job Name:</label>
+                    <input type="text" id="jobName" className='jp-vdk-input' placeholder={this.props.jobName} onChange={this._onNameChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobTeam">Job Team:</label>
+                    <input type="text" id="jobTeam" className='jp-vdk-input' placeholder="jobs" onChange={this._onTeamChange} />
+                </div>
+                <div className='jp-vdk-input-wrapper'>
+                    <label className='jp-vdk-label' htmlFor="jobRestApiUrl">Rest Api URL:</label>
+                    <input type="text" id="jobRestApiUrl" className='jp-vdk-input' placeholder="rest-api-url" onChange={this._onRestApiUrlChange} />
+                </div>
+            </>
+        );
+    }
+     /**
+   * Callback invoked upon changing the job name input.
+   *
+   * @param event - event object
+   */
+      private _onNameChange = (event: any): void => {
+        const nameInput = event.currentTarget as HTMLInputElement;
+        let value = nameInput.value;
+        if (!value) {
+            value = this.props.jobName;
+        }
+        sessionStorage.setItem("delete-job-name", value);
+    };
+    /**
+   * Callback invoked upon changing the job team input.
+   *
+   * @param event - event object
+   */
+    private _onTeamChange = (event: any): void => {
+        const teamInput = event.currentTarget as HTMLInputElement;
+        let value = teamInput.value;
+        if (!value) {
+            value = "default-team";
+        }
+        sessionStorage.setItem("delete-job-team", value);
```

Comment:
another way of approaching the description of session-stored values, that differs from documenting it,
is to implement a data structure(object) that describes the properties stored (self-explanatory for the code reader);
although I fancy the simplicity of direct key-value storage as well;

Reply:
updated here https://github.com/vmware/versatile-data-kit/issues/1484